gooddata-sdk/gooddata_sdk/table.py

```diff
@@ -193,12 +215,255 @@ def _as_table(response: ExecutionResponse) -> ExecutionTable:
     return ExecutionTable(response=response, first_page=first_page)
 
 
+@define
+class TableDimension:
+    item_ids: list[str]
```

Comment:
PythonSDK supports Python3.8; using `list` in data classes can cause issues. In that case, use `List`.